packages/strapi-admin/package.json

```diff
@@ -26,6 +26,8 @@
   },
   "dependencies": {
     "intl": "^1.2.5",
+    "react": "^16.8.1",
```

Comment:
Why is it added to the admin dependencies?

Reply:
No idea... I'll remove it.